.github/workflows/lint.yml

```
@@ -16,7 +16,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: "3.11"
```

```suggestion
          python-version: 3.11
```

Reason for suggestion commented below.

---

_Can_ keep it for without quotes for this and a few others, but think it looks nicer with all in quotes, as 3.10 needs to be in quotes.